docs/source/tutorials/qaoa.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def QAOAansatz(params, g=example_graph, circuit=False):
</pre><p>change an argument name for <code>circuit</code> ? as it is confusing, maybe <code>return_circuit</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>